.changeset/twenty-ravens-divide.md

```diff
@@ -0,0 +1,8 @@
+---
+"@spectrum-css/coachmark": minor
```

Comment:
Just making sure here: the changes in the coachmark CSS are actually a result of the breaking changes from popover. Would that still be considered a minor change in the coachmark CSS?